docs/src/pages/basics.mdx

```diff
@@ -173,6 +173,7 @@ function Scrolling() {
 Springs take pretty much everything, they don't just handle numbers.
 
 - Colors (names, rgb, rgba, hsl, hsla, gradients)
+- CSS Variables and their fallbacks (as long as they're attached to the `window` object)
```

Comment:
does this mean that vars not set on `#root` won't work?

Reply:
AFAIK you have to set them like:

```css
:root {
  --from: #3cffd0;
  --to: #277ef4;
}
```

How i've written it is obviously a bit unclear, do you have an idea how it might be better written?

Reply:
oh yeah, i meant `:root` 😅 . 

we could just replace '`window` object': "as long as they're declared on the `:root` pseudo-class."

or maybe: "CSS Variables (declared on `:root`) and their fallbacks."